backend/functions/host.json

```diff
@@ -4,8 +4,15 @@
     "applicationInsights": {
       "samplingSettings": {
         "isEnabled": true,
-        "excludedTypes": "Request"
+        "excludedTypes": ""
       }
+    },
+    "fileLoggingMode": "always",
+    "logLevel": {
+      "default": "Trace",
```

Comment:
Should the default level be `Trace`? Unless I'm misunderstanding, this is quite verbose, and could be outputting sensitive information.

https://learn.microsoft.com/en-us/azure/azure-functions/configure-monitoring?tabs=v2#configure-log-levels

Reply:
I set it to trace for initial ingestion, but Debug or Information levels would likely be a better option for us on our side.